packages/genymotion-bin/genymotion-bin.pacscript

```diff
@@ -14,5 +14,5 @@ maintainer="Arrowsome <ramtintoosi@gmail.com>"
 
 package() {
   sudo chmod +x "./genymotion-${pkgver}-linux_x64.bin"
-  sudo "./genymotion-${pkgver}-linux_x64.bin" -y -d "${pkgdir}/opt"
+  sudo "./genymotion-${pkgver}-linux_x64.bin" -y -d "/opt"
```

Comment:
Everything needs to be inside `${pkgdir}` for `package`.

Reply:
```
[Desktop Entry]
...
Icon=/usr/src/pacstall/genymotion-bin/opt/genymotion/icons/genymotion-logo.png
Exec=/usr/src/pacstall/genymotion-bin/opt/genymotion/genymotion
...
```

The desktop icon has invalid path in this case!

Reply:
Then `sed` it out during `package()`, such as `sed -i 's|/usr/src/pacstall/genymotion-bin||g' path/to/desktop/file`!